e2e/app/create-account-page.ts

```diff
@@ -154,7 +154,7 @@ export default class CreateAccountPage extends BasePage {
     const emailAddress = await Textbox.forLabel(this.page, {textContains: FIELD_LABEL.INSTITUTION_EMAIL, ancestorNodeLevel: 2});
```

Comment:
nit: I'd rename `emailAddress` to `emailAddressTextBox` to distinguish it from the address itself.